kubeflow/Dockerfile

```diff
@@ -2,9 +2,20 @@ ARG COMMON_IMAGE=
 FROM ${COMMON_IMAGE}
 MAINTAINER Thingpedia Admins <thingpedia-admins@lists.stanford.edu>
 
+ARG ADD_BOOTLEG=false
+RUN echo ${ADD_BOOTLEG}
+ARG BOOTLEG_VERSION=master
+RUN if [ ${ADD_BOOTLEG} == true ]; then \
+		git clone https://github.com/Mehrad0711/bootleg.git /opt/bootleg/ ; \
+		cd /opt/bootleg/ ; \
+		git checkout ${BOOTLEG_VERSION} && pip3 install --use-feature=2020-resolver -r requirements.txt && pip3 install --use-feature=2020-resolver -e . ; \
+	fi
+
+# next line needed to allow version bumps in torch and transformers take effect
+RUN rm -rf /opt/genienlp/
```

Comment:
This line is unnecessary, and breaks the use of the base image: if you bump the version requirement of torch in genienlp, the call to pip install will install the newer version.

Reply:
yeah but apparently it doesn't do so for transformers and torch. I couldn't figure why. Maybe it's a combination of package requirements that doesn't allow pip to upgrade them. I will omit this for now and try to figure later.